content/en/docs/tasks/manage-daemon/update-daemon-set.md

```diff
@@ -11,6 +11,8 @@ This page shows how to perform a rolling update on a DaemonSet.
 
 ## {{% heading "prerequisites" %}}
 
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```

Comment:
```suggestion
{{< include "task-tutorial-prereqs.md" >}}
```
You don't need to check your version of Kubernetes and also the front matter doesn't specify a minimum cluster version.

Reply:
Thanks. Updated.